src/core/ledger/transaction.ts

```diff
@@ -321,7 +321,7 @@ export const voidTransactions = (journalId, reason) => {
 }
 
 export const settlePayment = async (hash) => {
-  const result = await Transaction.updateMany({ hash }, { pending: false })
+  const result = await Transaction.updateMany({ hash, pending: true }, { pending: false })
```

Comment:
I guess it was implicit?

Reply:
ie: `nModified` would be 0 if `pending` was already `false`?

Reply:
fixed